optimum/onnxruntime/quantization.py

```diff
@@ -323,7 +323,7 @@ def quantize(
             per_channel=quantization_config.per_channel,
             mode=quantization_config.mode,
             weight_qType=quantization_config.weights_dtype,
-            input_qType=quantization_config.activations_dtype,
+            activation_qType=quantization_config.activations_dtype,
```

Comment:
In order for us to support `onnxruntime` version lower than `1.13.1`, could you adapt the `quantizer_factory` behavior depending on which `onnxruntime` version is used ?

Reply:
You're absolutely right, I just added it!